### PR TITLE
New answers!

### PR DESCRIPTION
**Pull request by fazo96**
Also fixed a few whitespace and small formatting issues

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -100,8 +100,20 @@ __IPFS is peer to peer__ because every node can use other nodes to get data whil
 
 __IPFS is also fully distributed__ because there is no central point of failure. The only non distributed part of the system is the bootstrap node list. The bootstrap node list is the list of nodes that IPFS connects to when it first starts, because it doesn't know anyone else yet! Then, nodes exchange their peer list, and more and more machines are discovered and connected together, strengthening the network. IPFS can remember nodes it connected to in the past, and the bootstrap list can be extended or changed, so it's almost impossible that your node can't get online. IPFS is also able to automatically find other computers running a node in local networks, so __it works even WITHOUT the internet!__
 
+<<<<<<< 4c4dfdd873a3b18657eb85c895313678a41b1756
```
Comment (simonv3) [inline README.md]: Accidentally left in?

Comment (fazo96) [inline README.md]: yes, taking it out now!

**Comment by RichardLitt** [comment]
Looks good to me!

**Comment by RichardLitt** [comment]
:+1: Thanks!

**Pull request merged**